jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitSelectionPanel.java

```diff
@@ -42,7 +42,20 @@
 
     private final SamProjectGenerator generator;
 
+    /**
+     * Takes a runtime name and filters based on it.
+     */
+    private interface RuntimeFilter {
+        boolean filterExpression(String runtime);
+    }
+
+    private static final RuntimeFilter filterNoRuntimes = (s) -> true;
```

Comment:
Consider renaming to `includeAllRuntimes` to increase clarity by removing a negative from the name.

Reply:
That's a better name, it used to be `filterNone` which is even less clear. Also added a changelog entry.